src/containers/StructurePage/resourceComponents/ApproachingRevisionDate.tsx

```diff
@@ -8,30 +8,45 @@
 
 import React, { useEffect, useState } from 'react';
 import styled from '@emotion/styled';
-import { colors } from '@ndla/core';
 import addYears from 'date-fns/addYears';
 import isBefore from 'date-fns/isBefore';
 import { IArticle } from '@ndla/types-draft-api';
 import Tooltip from '@ndla/tooltip';
 import { useTranslation } from 'react-i18next';
+import { Time } from '@ndla/icons/common';
 import { fetchDrafts } from '../../../modules/draft/draftApi';
 import handleError from '../../../util/handleError';
 import { getExpirationDate } from '../../ArticlePage/articleTransformers';
 
-const StyledWrapper = styled.div`
-  color: ${colors.white};
+const Wrapper = styled.div`
+  width: 24px;
+  height: 24px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const StyledIcon = styled.div`
   // TODO: Update when color is added to colors
-  background-color: #c77623;
+  background-color: transparent;
+  border: 1px solid #c77623;
   width: 20px;
   height: 20px;
   border-radius: 50%;
   display: flex;
   justify-content: center;
   align-items: center;
+  color: #c77623;
+`;
+
+const StyledTimeIcon = styled(Time)`
+  width: 24px;
+  height: 24px;
+  color: #c77623;
```

Comment:
Dette var en litt skummel farge. Burde den være i frontend-packages?

Reply:
Sjekk TODO øverst. Denne er definert som Oppgaver Mørk i oppdatert Figma for komponentbibliotek, men vi har ikkje fått dette overlevert.